### PR TITLE
consensus/clique: remove redundant pair of parentheses

### PR DESCRIPTION
**Pull request by yutianwu**

### CODE REVIEW

File: consensus/clique/clique.go [hunk 1]
```diff
@@ -26,6 +26,9 @@ import (
 	"sync"
 	"time"
 
+	lru "github.com/hashicorp/golang-lru"
+	"golang.org/x/crypto/sha3"
```
Comment (karalabe) [inline consensus/clique/clique.go:30]: The import order is whatever `goimports` gives us. I prefer that vs manual groups.

File: consensus/clique/snapshot.go [hunk 2]
```diff
@@ -22,12 +22,13 @@ import (
 	"sort"
 	"time"
 
+	lru "github.com/hashicorp/golang-lru"
```
Comment (karalabe) [inline consensus/clique/snapshot.go:25]: The import order is whatever `goimports` gives us. I prefer that vs manual groups.

File: consensus/clique/clique.go [hunk 3]
```diff
@@ -351,7 +352,7 @@ func (c *Clique) snapshot(chain consensus.ChainReader, number uint64, hash commo
 		headers []*types.Header
 		snap    *Snapshot
 	)
-	for snap == nil {
+	for {
```
Comment (karalabe) [inline consensus/clique/clique.go:355]: Although you are right, if nonetheless feel safer to leave this as is, it's a lot more future proof. It also makes the code more readable, as you can see that we're looping as long as `snap` is `nil`, vs. an infinite loop, for which you'd need to dig up the exit condition inside.

File: consensus/clique/snapshot.go (hunk 2 above)
Comment (yutianwu) [reply consensus/clique/snapshot.go:25]: the new import order is given by `goimports`. you can config `goimports` with `-local github.com/ethereum/go-ethereum`, that will give you a proper order.

Comment (fjl) [reply consensus/clique/snapshot.go:25]: Our policy is that we use what `goimports` does by default. We don't want to change all code to have three import 'groups'.

**Pull request merged**